code/datums/mind.dm

```diff
@@ -29,6 +29,10 @@ datum/mind
 	/// A list of every antagonist datum subordinate to this mind.
 	var/list/datum/antagonist/subordinate/subordinate_antagonists = list()
 
+	//Gang variables
+	var/obj/item/device/pda2/originalPDA //! The PDA that this crewmember started with - for gang PDA messages
+	var/gang_points = 0 //! The amount of points this mind has to redeem in lockers.
```

Comment:
Not great for this to be on mind, could just be an associative list on the gang datum? Or on the antagonist datum maybe

Reply:
yea could work